tools/istio-docker.mk

```diff
@@ -238,12 +247,12 @@ docker.citadel-test: $(ISTIO_DOCKER)/istio_ca.crt
 docker.citadel-test: $(ISTIO_DOCKER)/istio_ca.key
 	$(DOCKER_RULE)
 
-docker.node-agent: BUILD_ARGS=--build-arg BASE_VERSION=${BASE_VERSION}
+docker.node-agent: BUILD_ARGS=--build-arg BASE_VERSION=${BASE_VERSION} ${DOCKER_OPT}
 docker.node-agent: security/docker/Dockerfile.node-agent
 docker.node-agent: $(ISTIO_DOCKER)/node_agent
 	$(DOCKER_RULE)
 
-docker.node-agent-k8s: BUILD_ARGS=--build-arg BASE_VERSION=${BASE_VERSION}
+docker.node-agent-k8s: BUILD_ARGS=--build-arg BASE_VERSION=${BASE_VERSION} ${DOCKER_OPT}
```

Comment:
All these extra DOCKER_OPTS may be good for a prototype. They are not helpful for long term maintenance - and instead this BASE_OPT should be integrated into line 285.